fearless/Common/ViewController/SheetAlertViewLayout.swift

```diff
@@ -58,11 +60,24 @@ final class SheetAlertViewLayout: UIView {
         fatalError("init(coder:) has not been implemented")
     }
 
+    override func layoutSubviews() {
+        super.layoutSubviews()
+        closeButton.rounded()
```

Comment:
applyRoundedStyle() or round() or roundOff()

Reply:
closeButton is UIButton